samples/expert-finder-js/README.md

```diff
@@ -0,0 +1,92 @@
+---
+page_type: sample
+description: This sample demonstrates how to use a copilot plugin that can be used to search candidates for job. This sample can also be used as a search based messaging extension.
```

Comment:
This sample name should be: msgext-expert-finder and it should be linked to Root readme file as well.

Reply:
Summary should be something like below:

![image](https://github.com/user-attachments/assets/47d92a44-0d27-4ae6-8d7a-b511a1487d96)

Reply:
changed